Makefile

```diff
@@ -62,6 +62,10 @@ deploy: manifests kustomize
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
+# Download vendor dependencies
+vendor:
```

Comment:
Maybe add this target to `local-setup`

Reply:
It's added to `all`, that should work, no?